canonical/store-components
# Update filters component

## Done
- Updated `Filters` component to work with the beta store

## QA
- Go to https://store-components-49.demos.haus/?path=/story/filters--default-story
- Check that the `Filters` component renders
